### PR TITLE
convert Decimal Number to its Binary equivalent 

### PR DESCRIPTION
**Pull request by aditim26**
## Related Issue
-  Program for decimal to binary conversion using stack(#1055)

- A decimal number can be converted to its binary equivalent by dividing the number by 2 till it can be no more divided and then printing its remainder in the reverse order.

- For example:
image

## Proposed Changes
- Added the program for issue #1055 
- Changes in the Readme file

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

In this PR I have added a program for converting any decimal number to its binary equivalent 

This PR is solving issue #1055 which was created by me.

Examples of the test cases are given below 

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1   | 
Code                
image

|Screenshot #2    |
Output              
image

### CODE REVIEW

**Comment by aditim26** [comment]
@siddhi-244 Ma'am please review my PR

**Comment by Mrjoy832** [comment]
> @siddhi-244 Ma'am please review my PR

rather than taking a,b,c,x taking some proper variable name

**Comment by aditim26** [comment]
@Mrjoy832 I have made the required changes please check

**Comment by Mrjoy832** [comment]
Your readme is not following any guidelines ...image is also not viewable

You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Comment by aditim26** [comment]
But sir the image is visible from my side 
image

**Comment by aditim26** [comment]
sir I have added the Image but I dont know why its not visible 
image

**Comment by Mrjoy832** [comment]
> sir I have added the Image but I dont know why its not visible 
> image
> 

While taking the screnshot copy the image and just paste in here without any code syntax

**Comment by aditim26** [comment]
image

**Comment by Mrjoy832** [comment]
not here paste into just readme in same way

**Comment by aditim26** [comment]
Sir Please check if now its visible

**Comment by Mrjoy832** [comment]
> Sir Please check if now its visible

image
Not visible

**Comment by aditim26** [comment]
Sir please review the changes now

**Comment by Mrjoy832** [comment]
> Sir please review the changes now

Nice.

**Comment by Mrjoy832** [comment]
> > Sir please review the changes now
> 
> Nice...referred to admin to merge ...work on another issue.

File: S-Stack/Decimal_to_Binary.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include <iostream>
```
Comment (Lakhankumawat) [inline S-Stack/Decimal_to_Binary.cpp:1]: Rename file to `DecimalToBinary.cpp`

File: S-Stack/README.md [hunk 2]
```diff
@@ -104,7 +109,21 @@ Note that an input string is valid if:
...
+
+![image](https://user-images.githubusercontent.com/91279248/162582678-e84410dd-d018-4bca-8b52-630411a8cde5.png)
+
+## Algorithm for decimal to binary conversion:
+
+1. Divide the number by 2 and store its remainder
+2. Repeat step 1 unitl the number cannot be divided any further
+3. Print the remainder in reverse order
 
+**Time Complexity** is O(n)
```
Comment (Lakhankumawat) [inline S-Stack/README.md:129]: You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by Mrjoy832** [comment]
You have too many commits....I am Closing this PR, create new fresh PR on same issue with less commits.

**Pull request closed without merging**